json_serializable/lib/src/generator_helper.dart

```diff
@@ -221,9 +234,10 @@ class _GeneratorHelper {
   void _writeWrapper(Iterable<FieldElement> fields) {
     _buffer.writeln();
     // TODO(kevmoo): write JsonMapWrapper if annotation lib is prefix-imported
-    _buffer.writeln('''class $_helpClassName extends \$JsonMapWrapper {
-      final $_mixClassName _v;
-      $_helpClassName(this._v);
+    _buffer
+        .writeln('''class ${_wrapperClassName(true)} extends \$JsonMapWrapper {
```

Comment:
with `writeln` you're going to get an extra newline and this already ends in a newline. Do we need to?

Reply:
Honestly, dartfmt handles most of this. Not going to modify this in this CL